scripts/utils/bdlClassnameManager

```diff
@@ -29,12 +29,12 @@ const writeFile = promisify(fs.writeFile);
 */
 
 const conversionMap = {
-    // 'btn': 'bdl-Button',
-    // 'btn-group': 'bdl-ButtonGroup',
-    // 'btn-plain': 'bdl-Button--plain',
-    // 'btn-primary': 'bdl-Button--primary',
-    // 'is-disabled': 'bdl-is-disabled',
-    // 'label': 'bdl-Label',
+    btn: 'bdl-Button',
+    'btn-group': 'bdl-ButtonGroup',
+    'btn-plain': 'bdl-Button--plain',
```

Comment:
BTW, `PlainButton` component, doesn't use `Button` component (something we should eventually fix). So this can't be a modifier atm.

Reply:
@phora Would you recommend changing this to `bdl-PlainButton` or keeping `btn-plain` for now?

Reply:
bdl-PlainButton i would say